DropDownMenu.swift

```diff
@@ -117,7 +122,18 @@ open class DropDownMenu : UIView, UITableViewDataSource, UITableViewDelegate, UI
 		super.layoutSubviews()
         let contentHeight = menuCells.reduce(0) { $0 + $1.rowHeight }
 		
-		menuView.frame.size.height = contentHeight
+        //  Set the right height for tableView
+        //
+        //  If the total from the cells is less thant the height of the screen
+        //  then we keep all cells height
+        //  Else we take keep the the height of the screen so we can scroll from all its elements
+        
+        if contentHeight <= newTableViewFrame.size.height {
+            menuView.frame.size.height = contentHeight
+        }else{
+            menuView.frame.size.height = newTableViewFrame.size.height - 20 - 40 // -20 for statusbar & -40 for navigation bar
```

Comment:
` - 20 - 40` should be replaced by ` - visibleContentOffset`

The visible content offset is used to indicate the height occupied by decorations such as status bar and navigation bar inside `DropDownMenu.container`, when the menu is visible.